webview/src/experiments/components/table/header/TableHeaderCellContents.tsx

```diff
@@ -104,7 +104,7 @@ export const TableHeaderCellContents: React.FC<{
   setMenuSuppressed,
   resizerHeight
 }) => {
-  const isTimestamp = header.headerGroup.id === ColumnType.TIMESTAMP
+  const isTimestamp = header.id === 'Created'
```

Comment:
Funnily enough, there were two attempts in the code to move the timestamps icon menu to the right:

* uses `isTimestamp` and a `styles.moveToRight` class
* relies on `createdHeaderCell` class to move the icon menu to the right

The first way wasn't technically working so the second way was the one being applied to the actual cell. I decided to go the first way so I fixed the `isTimestamp` check and deleted the `createdHeaderCell` class.